### PR TITLE
[RPA-1224] Add rpaframework-actions

### PR DESCRIPTION
**Pull request by marcin-depcik**

### CODE REVIEW

File: rpaframework-actions/app/main.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    if args.minimizeWindowLocator:
+        locator = args.minimizeWindowLocator
+
+        actions.minimize_window(locator)
+
+    if args.maximizeWindowLocator:
+        locator = args.maximizeWindowLocator
+
+        actions.maximize_window(locator)
```
Comment (preludium) [inline rpaframework-actions/app/main.py]: make a switch instead

File: rpaframework-actions/src/automation/rpaframework.action-handler.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            case 'rpaFramework.maximizeWindow':
+                return this.maximizeWindow(request.input)
+            case 'rpaFramework.listWindows':
+                return this.listWindows()
+            default:
+                throw new Error('Action not found');
+        }
+    }
+
+    private async executableRunner(args: string[]) {
```
Comment (preludium) [inline rpaframework-actions/src/automation/rpaframework.action-handler.ts]: this does not look sounds like function name

File: rpaframework-actions/src/automation/rpaframework.action-handler.ts [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        const executablePath = process.env.RUNBOTICS_RPAFRAMEWORK_EXE_DIR;
```
Comment (preludium) [inline rpaframework-actions/src/automation/rpaframework.action-handler.ts]: rename this env

**Review by preludium — CHANGES_REQUESTED** [review]
packages generated by rush's symlinks do not work on the bot app. We must later create a different way to automatize package generation process

File: rpaframework-actions/app/main.py (hunk 1 above)
Comment (marcin-depcik) [reply rpaframework-actions/app/main.py]: args is a dict, so each if statement checks if the args property is not "None", I think in the case of a dictionary, the switch may not be the best solution

File: rpaframework-actions/src/tests/rpaframework.action-handler.test.ts [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    exec('calc', (error) => {
+        if (error) {
+            console.error(`Error executing command: ${error}`);
+            return;
+        }
+        console.log(`Calculator opened successfully`);
+    });
+};
+
+describe('RpaFrameworkActionHandler', () => {
```
Comment (preludium) [inline rpaframework-actions/src/tests/rpaframework.action-handler.test.ts]: outdated

File: rpaframework-actions/src/tests/rpaframework.action-handler.test.ts
Comment (preludium) [inline rpaframework-actions/src/tests/rpaframework.action-handler.test.ts:1]: file name is outdated

**Review by preludium — APPROVED** [review]
almost every file name is outdated

**Pull request merged**